jooq/src/main/kotlin/com/terraformation/backend/jooq/TerrawareGenerator.kt

```diff
@@ -33,7 +33,8 @@ class TerrawareGenerator : KotlinGenerator() {
       import com.terraformation.backend.db.EnumFromReferenceTable
       import org.jooq.impl.AbstractConverter
 
-    """.trimIndent())
+    """
+            .trimIndent())
```

Comment:
what's the reason for these extra spaces?

Reply:
is just from ktfmt?

Reply:
Yeah, this is ktfmt. Its handling of multiline strings leaves something to be desired.